src/microbe_stage/gui/PatchDetailsPanel.cs

```diff
@@ -595,6 +595,7 @@ private void MoveToPatchClicked()
             return;
         }
 
+        GetTree().CallGroup("ChemicalEquations", "UpdateEquation");
```

Comment:
I'm sorry to say but I don't like this solution. We don't use Godot groups for anything so adding it for just this one thing seems not good.

My preferred solution is still to trigger a GUI tooltip update in the editor in the patch changed callback. It needs a few levels of indirection but it shouldn't take that long to write all the proxy methods that get the notice to update where it is needed.

Reply:
Do we have some similar solution somewhere in the code? Would be easier to get inspired by it :D

Reply:
I can't immediately think of an exact match but `UpdateMPCost` in `CellEditorComponent.GUI.cs` has a loop that updates a property on all tooltips. So basically that but instead just calling a method on each tooltip that would then in turn call a method on all chemical equations that the tooltip contains (this is likely the only slightly tricky part as the tooltips need to be updated to know about the chemical equations inside them).